src/plugins/layout/stateprovider.ts

```diff
@@ -117,6 +106,12 @@ export function createTextLayoutState(textNode: Text): ITextLayoutState {
   return textState;
 }
 
+export function createIgnoreLayoutState(node: Node): IIgnoreLayoutState {
```

Comment:
I agree! Node type is already recorded 2 lines below. However, if ignored node was an Element, i will also add it's original tagName to the state (e.g. 'SCRIPT')

---
In reply to: [131241078](https://github.com/Microsoft/clarity-js/pull/25#discussion_r131241078) [](ancestors = 131241078)